Sources/NestKit/GitHub/GitHubURLBuilder.swift

```diff
@@ -10,7 +10,7 @@ public enum GitHubURLBuilder {
         url.appending(components: "releases", "download", version, fileName)
     }
 
-    static func assetURL(_ url: URL, version: GitVersion) throws -> URL {
+    static func assetURL(_ url: URL, version: GitVersion, hasExcludedVersion: Bool) throws -> URL {
```

Comment:
Whether a specific version is excluded or not is not a concern of `GitHubURLBuilder`.
This function should not know such caller side concerns.
So, how about renaming the argument to `onlyLatestVersion`?

Reply:
And the argument `hasExcludedVersion` is ignored on a case where `version` indicates a specific version.
Please write DocC explaining the case.

Reply:
Thanks!
You were absolutely right.
I've implemented `releaseAssetURL` to fetch the release list endpoint: https://github.com/mtj0928/nest/pull/33/commits/3d93849157361f32c97fa9c60dd04d68c1ea84c1
How does this look?